sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/exception/CosmosBadRequestException.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.exception;
+
+import org.springframework.lang.Nullable;
+
+public class CosmosBadRequestException extends CosmosAccessException {
```

Comment:
final?

Reply:
same for others

Reply:
@FabianMeiswinkel is there a reason to make it final?

Reply:
@trande4884 - general guidance from central team is to make public surface area final unless there is a reason to not do this. And I don't see any reason why we should expect customers to have to craeted extended sub classes. Also, making it final afterwards would always be breakign - while removing final is not a breaking change. So, default should be final unless there is a good reason to allow extending it. Makes sense?

Reply:
Makes sense I will change it

Reply:
Done.